UIComponents/Sources/UIComponents/Helpers/TapGestureRecognizer.swift

```diff
@@ -0,0 +1,29 @@
+import UIKit
+
+public class TapGestureRecognizer: UITapGestureRecognizer {
+    private var tapAction: ((UITapGestureRecognizer) -> Void)?
```

Comment:
Can we use let instead of var in that case? I think this property tends to be immutable.

Reply:
Done